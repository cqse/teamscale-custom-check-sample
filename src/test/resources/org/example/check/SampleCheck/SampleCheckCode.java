package org.example.check.SampleCheck;

public class SampleCheckCode {

	private void foo(int a) {
	}

	private void bar(int a, int b) {
	}

}