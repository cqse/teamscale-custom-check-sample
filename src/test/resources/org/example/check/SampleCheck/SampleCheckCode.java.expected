Lines 5-5: This is a sample finding for method foo.
Lines 8-8: This is a sample finding for method bar.
